src/main/java/ca/corefacility/bioinformatics/irida/ria/web/pipelines/PipelineController.java

```diff
@@ -471,7 +471,8 @@ private String localizedParamOptionLabel(Locale locale, String workflowName, Str
 
             String analysisDescription = parameters.getDescription();
             Boolean writeResultsToSamples = parameters.getWriteResultsToSamples();
-            Boolean emailPipelineResult = parameters.getEmailPipelineResult();
+            Boolean emailPipelineResultCompleted = parameters.getEmailPipelineResultCompleted();
+            Boolean emailPipelineResultError = parameters.getEmailPipelineResultError();
```

Comment:
Again kinda a weird logic thing here with the way this is named.  If you select the "error" dropdown, `emailPipelineResultError` is true.  If you select the "completion (error or success)" dropdown, `emailPipelineResultCompleted` is true.  The names of the variables would suggest though that with the 2nd dropdown "completion (error or success)" that both of these values would be true.  

If you want to keep it that only 1 of these will be true, I think it would be better set as an enum or something on `AnalysisSubmission` rather than the 2 boolean to indicate the "level" of email that will be sent here `None`, `Error`, `Completion or All or something`.

Reply:
Thanks for taking a look at that. I updated it so that both `email_pipeline_result_completed` and `email_pipeline_result_error` are set to true if the `completion (error or success)` option is selected. Also, updated the `required-data.sql`. I for some reason always forget about it. Updated in b3f3e8bf5be4b6ac25d3068a2b46b47f9672522d